{{ cookiecutter.service_name }}/internal/{{ cookiecutter.entity_name }}/{{ cookiecutter.entity_name }}.go

```diff
@@ -0,0 +1,7 @@
+package {{ cookiecutter.entity_name }}
+
+type Service interface {
+}
+
+type Repository interface {
+}
```

Comment:
Add a comment with and example of the declaration of a function for the repository